build.sbt

```diff
@@ -56,6 +56,26 @@ lazy val commonSettings = Seq(
       if (sys.env.getOrElse("CI", "false") == "true") "-Xfatal-warnings" else ""),
   libraryDependencies ++= commonDeps)
 
+lazy val assemblySettings = Seq(
```

Comment:
Actually! pls add a reference to https://github.com/sbt/sbt/issues/2255 + a short explanation in comment here

Reply:
Sure - comment in file added. Short explanation is in the PR description, but here we go again :)
- as usage of https://github.com/pathikrit/better-files has been added to the runtime logic of the codec-registration-checker-compiler-plugin - this dependency has to be provided within a fat jar when ASH gets published - hence changes inside build.sbt
TL;DR: without publishing fat jar our commercial project (Hydra) did not find classes better-files during compilation - that's why fat-jar was needed

Reply:
Yes! I do get it... I meant a comment **in code**, coz locating comments on GitHub via `git blame` isn't particularly convenient :/

Reply:
Assume someone would revisit this place in, say, 3 years 😅

Reply:
Ok, so - is the comment in build.sbt enough - or write some more detailed explanation?

Reply:
done